File: VB/DXWebApplication1/Controllers/HomeController.vb
Imports System
Imports System.Web.Mvc
Imports DevExpress.Web.Mvc
Imports DevExpress.XtraRichEdit
Imports DXWebApplication1.Models

Namespace DXWebApplication1.Controllers
	Public Class HomeController
		Inherits Controller

		Public Function Index() As ActionResult

            Return View()
        End Function

        Public Function RichEditPartial() As ActionResult
            Dim model = New RichEditData() With {.DocumentId = Guid.NewGuid().ToString(), .DocumentFormat = DocumentFormat.Rtf, .Document = DataHelper.GetDocument()}
            Return PartialView(model)
        End Function
    End Class
End Namespace

File: CS/DXWebApplication1/Controllers/HomeController.cs
using System;
using System.Web.Mvc;
using DevExpress.Web.Mvc;
using DevExpress.XtraRichEdit;
using DXWebApplication1.Models;

namespace DXWebApplication1.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return View();
        }

        public ActionResult RichEditPartial() {
            var model = new RichEditData()
            {
                DocumentId = Guid.NewGuid().ToString(),
                DocumentFormat = DocumentFormat.Rtf,
                Document = DataHelper.GetDocument()
            };
            return PartialView(model);
        }
    }
}